### PR TITLE
Task/6 login page

### PR DESCRIPTION
**Pull request by whiteNsky**
#6

### UPDATE V1.0.1
Changes:
- [x] Added togglePassword
- [x] Fixed eye-icon image positioning
- [x] Changed the font style of login-forms
- [x] Resized header to 1.5px;

### Problem
- Makita ang eye-icon maskig wala pa ni type ang user og password

### Solution

- [x] Added v-if="password.length > 0" to hide the eye-icon

### CODE REVIEW

File: components/LoginPage.vue [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+          </div>
+
+          <button type="submit" class="login-button">LOG IN</button>
+        </form>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
```
Comment (Techies-John) [inline components/LoginPage.vue:52]: use <script setup lang="ts">, try researching more about vue compositions api

File: components/LoginPage.vue [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+              <input
+                :type="showPassword ? 'text' : 'password'"
+                id="password"
+                v-model="password"
+                class="form-control"
+                placeholder=""
+                required
+              />
+              <div class="eye-icon" @click="togglePassword">
+                <img :src="showPassword ? '/assets/static-images/eye-open.png' : '/assets/static-images/eye-close.png'" id="eyeicon" />
```
Comment (Techies-John) [inline components/LoginPage.vue:34]: use computed()

**Review by sggenDebugs — DISMISSED** [review]
Okay looks good. 👍🏿🖤

File: components/LoginPage.vue [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+          <div class="form-group">
+            <label for="email">Silliman Email</label>
+            <input type="email" id="email" v-model="email" class="form-control" placeholder="" required />
+          </div>
+
+          <div class="form-group">
+            <label for="password">Password</label>
+            <div class="password-input-wrapper">
+
+              <input :type="showPassword ? 'text' : 'password'" id="password" v-model="password" class="form-control"
```
Comment (Techies-John) [inline components/LoginPage.vue:19]: No logics in template. Use computed.

File: components/LoginPage.vue [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+                placeholder="" required />
+
+              <div class="eye-icon" v-if="password.length > 0" @click="togglePassword">
+                <img :src="showPassword ? eyeOpen : eyeClose" id="eyeicon" />
```
Comment (Techies-John) [inline components/LoginPage.vue:23]: Use computed

File: components/LoginPage.vue [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+              </div>
+            </div>
+            <div class="password-strength-bar">
+              <div :class="['strength-indicator', passwordStrengthClass]"
+                :style="{ width: passwordStrengthPercentage + '%' }"></div>
```
Comment (Techies-John) [inline components/LoginPage.vue:28]: use computed for :class and :style

**Review by Techies-John — CHANGES_REQUESTED** [review]

**Review by Techies-John — APPROVED** [review]

**Review by sggenDebugs — APPROVED** [review]
okay looks nice 👍🏿👍🏿🖤🖤

**Pull request merged**